ambari-infra/ambari-infra-solr-client/src/main/java/org/apache/ambari/infra/solr/AmbariSolrCloudCLI.java

```diff
@@ -144,6 +146,11 @@ public static void main(String[] args) {
       .desc("Remove AdminHandlers request handler from solrconfig.xml (command)")
       .build();
 
+    final Option transferZnodeOption = Option.builder("tz")
```

Comment:
These `Option`s should be reused below for`cli.hasOption` and `cli.getOptionValue`, instead of repeating the strings.

Reply:
i think it's more like a global issue